src/sysfiles.c

```diff
@@ -1642,6 +1642,11 @@ Int SyGetch (
 UInt   syCTRO;                          /* number of '<ctr>-O' pending     */
 UInt   syESCN;                          /* number of '<Esc>-N' pending     */
 
+Obj StandardInput = 0;
+Obj StandardOutput = 0;
+Obj StandardError = 0;
+
```

Comment:
These are currently unused, I seem to remeber imagining having to access these streams from the kernel level.

Reply:
So, do you mean you want to keep them "in case", or that you will update this PR to drop them?!?

If you intend to keep them, please add at least a tiny comment.

Reply:
The best outcome would be to not need them at the C level at all, so maybe I remove them right now, and only add them if there's a really good reason to have them.